maquetado/application.css

```diff
@@ -0,0 +1,4 @@
+.title {
```

Comment:
add this file to the gitignore.
https://trello.com/c/lBiCscbp/11-setup-del-proyecto